Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -273,7 +273,7 @@
     <!-- XTXT: find details about exposures in the contact journal - increased risk -->
     <string name="risk_details_find_details_in_journal_increased_risk">Weitere Informationen zu Ihren Risiko-Begegnungen finden Sie in Ihrem Kontakt-Tagebuch.\n\nSie haben ein erhöhtes Risiko, sich angesteckt zu haben. Falls Sie infiziert sind, können Sie Ihrerseits andere Personen anstecken, auch wenn Sie geimpft oder genesen sind, und auch, wenn Sie keine Symptome haben.</string>
     <!-- XMSG: risk details - link to faq, something like a bullet point -->
-    <string name="risk_details_increased_risk_faq_link_text">"Lassen Sie sich testen, auch wenn Sie keine Symptome haben. Bei einer Warnung über ein erhöhtes Risiko („Rote Kachel“) haben Sie Anspruch auf einen kostenlosen Test. Dies muss nicht notwendigerweise ein PCR-Test sein."</string>
+    <string name="risk_details_increased_risk_faq_link_text">"Lassen Sie sich testen, z.B. mit einem Antigen-Schnelltest, oder testen Sie sich selbst, am besten mehrfach im Abstand von zwei Tagen. Sollte ein Test positiv ausfallen, lassen Sie diesen durch einen PCR-Test absichern."</string>
```

Comment:
According to [DIN 5008](https://de.wikipedia.org/wiki/DIN_5008#Abk%C3%BCrzungen_(Abschnitt_4.5)) `z.B.` should be written with a space e.g. `z.&nbsp;B.`

Reply:
Currently we use the plain z.B. without &nbsp; throughout the whole text files, so I wouldn't change that now in this pull request

Reply:
I agree that it would not be good to introduce inconsistencies in this PR even if the orthography is incorrect according to DIN rules.